contrib/tetragon-rthooks/cmd/setup/main.go

```diff
@@ -24,16 +24,19 @@ const (
 )
 
 type Install struct {
-	Interface           string `default:"oci-hooks" enum:"oci-hooks" help:"Hooks interface (${enum})"`
-	LocalBinary         string `default:"/usr/bin/tetragon-oci-hook" help:"Source binary path (in the container)"`
-	LocalInstallDir     string `required help:"Installation dir (in the container)."`
-	HostInstallDir      string `required help:"Installation dir (in the host). Used for the binary and the hook logfile."`
-	FailAllowNamespaces string `help:"Comma-separated list of namespaces to allow Pod creation for, in case tetragon-oci-hook fails to reach Tetragon agent."`
-	Daemonize           bool   `help:"Daemonize install command. If a termination signal is send, the process will remove the files it installed before exiting"`
+	Interface       string `default:"oci-hooks" enum:"oci-hooks" help:"Hooks interface (${enum})"`
+	LocalBinary     string `default:"/usr/bin/tetragon-oci-hook" help:"Source binary path (in the container)"`
+	LocalInstallDir string `required help:"Installation dir (in the container)."`
+	HostInstallDir  string `required help:"Installation dir (in the host). Used for the binary and the hook logfile."`
+	Daemonize       bool   `help:"Daemonize install command. If a termination signal is send, the process will remove the files it installed before exiting"`
 
 	OciHooks struct {
 		LocalDir string `default:"/hostHooks" help:"oci-hooks drop-in directory (inside the container)"`
 	} `embed:"" prefix:"oci-hooks."`
+
+	HookArgs struct {
+		Args []string `arg:"" optional:""`
```

Comment:
fyi: I had to discover kong to understand that `arg:""` would mean this is a positional argument and that all the rest would be added to this slice.

Reply:
The trick is actually in the `passthrough:` of the HookArgs struct.